src/Playwright/Waiter.cs

```diff
@@ -53,7 +55,7 @@ internal void Log(string log)
         {
             _logs.Add(log);
             var logArgs = new { info = new { waitId = _waitId, phase = "log", message = log } };
-            Forget(_channel.Connection.SendMessageToServerAsync(_channel.Guid, "waitForEventInfo", logArgs));
+            _ = _channel.Connection.SendMessageToServerAsync(_channel.Guid, "waitForEventInfo", logArgs);
```

Comment:
waitForResponse: false?

Reply:
We don't want the connection to keep track of the response because we don't care about it.